### DISCUSSION TITLE
z2m stability issues

### DISCUSSION THREAD

**Discussion by mackowskim** [Koenkk/zigbee2mqtt#25138] [General]
Hi,
Is it only me, having stability issues since v1,4* through v2.0? 
I'm not entirely sure if it was HA update or z2m itself, but I started observing huge lags on the zigbee network, which worked with no bigger issues before.

Sometimes the lag between swiching and light change reaches 10 sec.
I'm getting timeouts all around, pairing issues and even NWK table full.

`z2m: Failed to interview '0xa4c1388cf3abc1ec', device has not successfully been paired
error 2024-12-09 15:16:11z2m: Publish 'get' 'state' to '0xa4c1388cf3abc1ec' failed: 'Error: ZCL command 0xa4c1388cf3abc1ec/1 genOnOff.read(["onOff"], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Timeout - 18542 - 1 - 235 - 6 - 1 after 10000ms)'`

`z2m: Publish 'set' 'state' to 'Led kuchnia blat' failed: 'Error: ZCL command 0xa4c1389e0775a597/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (--> 'SREQ: ZDO - extRouteDisc - {"dstAddr":52012,"options":0,"radius":30}' failed with status '(0xc7: NWK_TABLE_FULL)' (expected '(0x00: SUCCESS)'))'`

I did rebuild the network on 1.4 (I was the lucky one) and then the issues started.
Now I'm on 2.0 beta (z2m edge) and not much better.

My zigbee coordinator is slzb-06p10 on channel 11 and 2.4GHz WiFi is also on channel 11 (so it should not overlap).

Any ideas, suggestions or the same experience from anyone?

**Comment by IceColdCrash** [comment]
Having exactly the same issue with slzb-06p7,
My Zigbee is on channel 25 and wifi in channel 1 so not the issue,
Either it was the Z2M update or HomeAssistant core update,
Some devices sort go into some sort of "sleeping" take a while ti respond and then work fine
